apps/ledger-live-desktop/src/renderer/hooks/useDeeplinking.ts

```diff
@@ -284,6 +284,9 @@ export function useDeepLinkHandler() {
         case "recover-restore-flow":
           navigate("/recover-restore");
           break;
+        case "exchange":
+          navigate(`/exchange/${path}`, undefined, search);
```

Comment:
do not we need to redirect to buy-sell basic app when "path" is undefined?

Reply:
this is redundant, I'll remove, it was added when spiking deep-linking.